README.md

```diff
@@ -52,12 +60,18 @@ docker run \
     ghcr.io/martinohansen/ynabber:latest
 ```
 
-Or deploying to Kubernetes with kubectl:
+## Readers
 
-```bash
-kubectl create configmap ynabber-env --from-env-file=ynabber.env
-kubectl apply -f kubernetes.yaml
-```
+Currently supported readers and verified banks
+
+| Reader   | Bank            |   |
+|----------|-----------------|---|
+| Nordigen | NORDEA_NDEADKKK | ✅ |
+
+Note that additional banks may also be supported, but they have not been
+specifically verified and tested yet. Please open an
+[issue](https://github.com/martinohansen/ynabber/issues/new) if you have
+problems with a specific bank.
 
```

Comment:
Is this addition in README actually necessary?

I think there is nothing NORDEA_NDEADKKK specific right now in the source code.
And although there are no automatic tests for banks I use, I can verify that it also works for:
ALANDSBANKEN_AABAFI22
NORDEA_NDEAFIHH
NORWEGIAN_FI_NORWNOK1
S_PANKKI_SBANFIHH

Although, perhaps I should test it for this PR first.